tests/Database/DatabaseQueryBuilderTest.php

```diff
@@ -2571,16 +2571,18 @@ public function testFirstMethodReturnsFirstResult()
     public function testPluckMethodGetsCollectionOfColumnValues()
     {
         $builder = $this->getBuilder();
-        $builder->getConnection()->shouldReceive('select')->once()->andReturn([['foo' => 'bar'], ['foo' => 'baz']]);
-        $builder->getProcessor()->shouldReceive('processSelect')->once()->with($builder, [['foo' => 'bar'], ['foo' => 'baz']])->andReturnUsing(function ($query, $results) {
+        $builder->getConnection()->shouldReceive('setFetchAllArgs', 'resetFetchAllArgs')->once();
+        $builder->getConnection()->shouldReceive('select')->once()->andReturn(['bar', 'baz']);
```

Comment:
The reason these tests have changed is because now the query returns the format:
```php
[
    'selected-key-value' => 'selected-column-value',
    // ...
]
```

Whereas previously, the format came back differently which had to be keyed manually:
```php
[
    [
        'selected-key' => 'selected-key-value',
        'selected-column' => 'selected-column-value',
    ],
    // ...
]
```